lib/vmpooler/statsd.rb

```diff
@@ -0,0 +1,12 @@
+require 'rubygems' unless defined?(Gem)
+
+module Vmpooler
+  class Statsd
+    def initialize(
+      s = 'statsd',
+      port = 8125
+    )
+      @server = Statsd.new s, port
```

Comment:
Consider wrapping the arguments to `Statsd#new` in parenthesis if you're using the return value in an assignment like this. It's functionally identical but it'll probably help avoid having to take a moment to parse the spaces here if a person has to do something here in the future.